CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,55 @@
+# Fabric Code of Conduct
+
+We are committed to providing a friendly, safe and welcoming environment for all, regardless of level of experience, gender identity and expression, sexual orientation, disability, personal appearance, body size, race, ethnicity, age, religion, nationality, or other similar characteristic.
```

Comment:
While both are generally social constructs, the general definition I'm used to is that race refers to biology and physical characteristics (hair texture, skin colour, etc) while ethnicity refers to things like cultural expression and identification.

I don't really think the inclusion of either term here will be problematic, personally — do you have any specific concerns?

Reply:
If you are talking about physical characteristics, that should fall under personal appearance in my opinion.
Regarding my concerns, I can quote the [relevant wikipedia article](https://en.wikipedia.org/wiki/Race_(human_categorization)):

> Since the second half of the 20th century, the association of race with the discredited theories of scientific racism has contributed to race becoming increasingly seen as a largely pseudoscientific system of classification. Although still used in general contexts, race has often been replaced by less ambiguous and loaded terms: populations, people(s), ethnic groups, or communities, depending on context.

In French and other latin languages, the word "racism" is notably defined as "believing in the existence of human races". So I think you can see why the word can be perceived as offensive by people from those cultures.

Reply:
Interesting, okay, I wasn't aware of this.

I think it's useful to tell people that they can't discriminate based on race, purely because there _are_ people that still use and understand the term in that manner — although it may be worth pointing out that `racism` as a specific term isn't something we've directly used in our documentation.

We could replace the term with culture/ethnicity across the documentation, perhaps — I'll have a chat with the others

Reply:
Having chatted about this and done some research, I think what I'm going to do is keep the references to `race` (as a social term, rather than a scientific term) and clarify it by including mentions of culture across the documentation.

At the end of the day, this is an English-language community and people are 100% welcome to come and chat to us if they have an issue with the policies, so we can always explain it to them if they get the wrong end of the stick.

I'm happy to chat more about this, but I believe that removing the term altogether detracts from the purpose of this documentation and may confuse people that are looking for explicit condemnation of racial discrimination.

Thanks for your feedback, though!